src/netbase.cpp

```diff
@@ -651,7 +667,7 @@ void CNetAddr::SetRaw(Network network, const uint8_t *ip_in)
             memcpy(ip, ip_in, 16);
             break;
         default:
-            assert(!"invalid network");
+	    assert(true);  // invalid network
```

Comment:
I think this changes the behavior. Before, it would fail printing the message "invalid network", but this is essentially a no-op.